lib/matplotlib/tests/test_textpath.py

```diff
@@ -0,0 +1,42 @@
+from copy import copy, deepcopy
+
+from numpy.testing import assert_array_equal
+from matplotlib.textpath import TextPath
+
+
+def test_set_size():
```

Comment:
Can this get a docstring, even if very cursory?  I can't really tell what it is supposed to be testing...

Reply:
I have added a simple docstring. I have revisited the test and I think that it tests the required functionality.